### PR TITLE
 Update the documentation

### PR DESCRIPTION
**Pull request by Aitical**
update the old doc for the latest code

### CODE REVIEW

File: docs/io.md [hunk 1]
```diff
@@ -15,17 +15,17 @@ data = mmcv.load('test.yaml')
 data = mmcv.load('test.pkl')
 # load data from a file-like object
 with open('test.json', 'r') as f:
-    data = mmcv.load(f)
+    data = mmcv.load(f, file_format='json')
```
Comment (hellock) [inline docs/io.md]: This was intended. Just to show the common usage without explicit `file_format` arguments.

**Comment by hellock** [comment]
Thanks for the fix. Just a minor comment.

**Comment by Aitical** [comment]
Ok, get it.

**Comment by hellock** [comment]
Oh you misunderstood my intension. The last two changes are ok.

**Pull request merged**